rero_ils/modules/operation_logs/cli.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2021 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Click command-line interface for operation_log record management."""
+
+from __future__ import absolute_import, print_function
+
+import json
+
+import click
+from flask.cli import with_appcontext
+
+from rero_ils.modules.operation_logs.api import OperationLog
+from rero_ils.modules.operation_logs.models import OperationLogOperation
+from rero_ils.modules.utils import extracted_data_from_ref
+
+
+@click.command('migrate_virtua_operation_logs')
+@click.option('-f', 'input_file', help='Logs input file')
+@with_appcontext
+def migrate_virtua_operation_logs(input_file):
+    """Migrate Virtua operation logs to REROILS."""
+    with open(input_file, 'r', encoding='utf-8') as request_file:
```

Comment:
If the file is always needed it is better to make it a parameter:
`@click.argument('input_file', type=click.File('r'))`Click can open the file already.
If the file can be big, it is better to implement also lazy loading for the json file.
Verbose should be nice to see an feedback if needed.